plugins/org.jboss.tools.openshift.cdk.server/src/org/jboss/tools/openshift/cdk/server/ui/internal/CDKServerWizardFragment.java

```diff
@@ -78,6 +80,7 @@ public Composite createComposite(Composite parent, IWizardHandle handle) {
 		credentials.addCredentialListener(new ICredentialCompositeListener() {
 			@Override
 			public void credentialsChanged() {
+				selectedUser = credentials.getUser();
```

Comment:
Line 123 is the initial value after creating the credential composite.   Line 83 is updating any time the credential composite fires an event. 

This is all necessary because performFinish is done in a non-UI thread, so cannot get the value of a widget due to invalid thread access. So the value must be cached in order to be used during performFinish.